doc/sphinx/language/gallina-extensions.rst

```diff
@@ -782,6 +782,7 @@ Section :ref:`gallina-definitions`).
 .. cmd:: Section @ident
 
    This command is used to open a section named `ident`.
```

Comment:
This line should be:
```This command is used to open a section named :token:`ident`.```
Would you mind fixing it at the same time?

Reply:
Done.